compiler/main/arg.cpp

```diff
@@ -182,6 +182,12 @@ void usage(const ArgumentState* state,
             break;
 
           case 'P':
+            if (desc[i].location != 0)
```

Comment:
consider documenting this on line 340, 'P' takes a `std::string*`

Reply:
I assume you mean in the list of "flag types"? (line 352 for me). Maybe I rebased and shifted it